localize.js

```diff
@@ -126,30 +134,40 @@ const strategies = {
 };
 
 const psuedoLocalizeString = (string, options = { strategy: "accented" }) => {
-  let opts = strategies[options.strategy];
+  let strategyOptions = strategies[options.strategy];
 
   let pseudoLocalizedText = "";
   for (let character of string) {
-    if (opts.map[character]) {
+    const convertedCharacter = strategyOptions.map[character]
+    const characterAsInt = parseInt(character)
```

Comment:
I think we can move this inline to line 146.

```js
!isNaN(parseInt(character))
```
imo it's still totally legible and we avoid executing parseInt for every character if `elongateNumbers` number is false.

Now that I think about it do we need to parseInt?

would 

```js
      options.elongateNumbers &&
      EVEN_NUMBER_MAP[character]
```
not be enough?